docs/model-cards/marin_8b_base.md

```diff
@@ -0,0 +1,156 @@
+---
+license: apache-2.0
+datasets:
+- allenai/dolmino-mix-1124
+- allenai/olmo-mix-1124
+language:
+- en
+---
+## Model Details
+
+<img alt="Marin Logo" src="https://huggingface.co/datasets/marin-community/blog-images/resolve/main/marin-boat.png" width="242px" style="margin-left:'auto' margin-right:'auto' display:'block'">
+
+
+# Model Card for Marin 8B Base
+
+This is the model card for the Marin 8B Base model. The Marin Project is a collaborative effort to develop open-source foundation models.
+
+
+Marin 8b was trained on a variety of datasets:
```

Comment:
Say base?  Also, can we make the model card for the specific version - marin-8b-base-deeper-starling?  I think we might have multiple 8Bs and it will be confusing if we don't version things properly.